src/systems/systemItemsDialog.component.tsx

```diff
@@ -31,10 +63,47 @@ const SystemItemsDialog = React.memo((props: SystemItemsDialogProps) => {
   const [parentSystemId, setParentSystemId] = React.useState<string | null>(
```

Comment:
I just noticed this isn't reset on close, so it doesn't go back to the value provided by the props. Could be a separate task as its a bug unaffected by this PR.